lib/common/typeorm.utils.ts

```diff
@@ -1,13 +1,33 @@
 import { Logger } from '@nestjs/common';
 import { Observable } from 'rxjs';
 import { delay, retryWhen, scan } from 'rxjs/operators';
-import { Connection, ConnectionOptions, EntityManager } from 'typeorm';
+import { Connection, ConnectionOptions, EntityManager, Repository, AbstractRepository } from 'typeorm';
 import * as uuid from 'uuid/v4';
 
```

Comment:
Add `type CustomRepository = Repository | AbstractRepository`

Reply:
Would adding the type def at the top of this file be the place for it?

Reply:
@JoeFitzgibbons 
I think there is no general rule where to put `type def` in NestJS. Therefore having it close to the source does make sense for me in this case.